engine/verification/utils/unittest/fixture.go

```diff
@@ -39,6 +39,11 @@ import (
 	"github.com/onflow/flow-go/utils/unittest"
 )
 
+const (
+	// TODO: enable parallel execution once cadence type checker bug is fixed.
```

Comment:
Not really a type checker bug - we need to change programs cache to ensure consistent use of pointers with enabled concurrent execution - type checker relies on that.

Reply:
bug / feature.  i don't really care what we label this.

imo, we should make try to make cadence reentrant safe, rather than changing program cache to ensure the cached object is only used by the interpreter that produces the object in the first place.